server/lib/db.py

```diff
@@ -48,18 +48,36 @@ class Club(Base):
     name = Column(String(250), nullable=False)
     school_id = Column(Integer, ForeignKey('school.id'))
     description = Column(String(1024), nullable=False)
+    img_type = Column(String(100), nullable=False)
     created_on = Column(DateTime, nullable=False, server_default=text("CURRENT_TIMESTAMP"))
 
     school = relationship(School)
 
+class Position(Base):
+    __tablename__ = 'position'
+    id = Column(Integer, primary_key=True)
+    position_type = Column(String(100), nullable=False, unique=True)
+    created_on = Column(DateTime, nullable=False, server_default=text("CURRENT_TIMESTAMP"))
 
 class UserToClubMapping(Base):
     __tablename__ = 'user_to_club_mapping'
     id = Column(Integer, primary_key=True)
     user_id = Column(Integer, ForeignKey('user.id'))
     club_id = Column(Integer, ForeignKey('club.id'))
-    is_admin = Column(Boolean, server_default=text("FALSE"))
     created_on = Column(DateTime, nullable=False, server_default=text("CURRENT_TIMESTAMP"))
 
     user = relationship(User)
     club = relationship(Club)
+
+class UserClubPositionMapping(Base):
```

Comment:
wasnt' this in another review? just delete that review

Reply:
I think this is unique to this PR